scripts/swupd_performance/swupd_logging_setup.bash

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/bash
+
+set -e
+
+DIRECTORY_PREFIX="/var/log/swupd"
+
+echo "Setup staging directory"
+
+if [ -d "$DIRECTORY_PREFIX"/staging ]; then
+	echo "Staging directory already exists" || true
```

Comment:
that `|| true` isn't really doing anything you can remove it

Reply:
Good catch. Thanks

Reply:
Done